lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -74,21 +74,21 @@ def self.included(includer)
           option :distro,
             :short => "-d DISTRO",
             :long => "--distro DISTRO",
-            :description => "Bootstrap a distro using a template. [DEPRECATED] Use -t / --bootstrap-template option instead.",
+            :description => "Bootstrap a distro using a template. [DEPRECATED] Use -b / --bootstrap-template option instead.",
             :proc        => Proc.new { |v|
               Chef::Log.warn("[DEPRECATED] -d / --distro option is deprecated. Use --bootstrap-template option instead.")
               v
             }
 
           option :bootstrap_template,
-            :short => "-t TEMPLATE",
-            :long => "--bootstrap-template TEMPLATE",
+            :short => "-b BOOTSTRAP_TEMPLATE",
```

Comment:
Personally, I would rather we change the short option flag for the winrm transport so that the common parameters between knife bootrstrap, and knife bootstrap winrm remain the same. Other than this one nit, everything looks good.